bee-common/bee-packable-derive/src/lib.rs

```diff
@@ -1,7 +1,7 @@
 // Copyright 2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-//! This crate provides the [`Packable`] derive macro.
```

Comment:
Any particular reason for getting rid of the doc linking?

Reply:
Same question

Reply:
yes, the link is pointing to the macro itself so it is misleading, technically `bee-packable` is not a dependency of `bee-packable-derive` so the `Packable` trait is not available.